src/components/Cards/DetailCard/auto.vue

```diff
@@ -129,8 +129,18 @@ export default {
     parseArrayValue(value, excludes, label) {
       if (Array.isArray(value)) {
         const tp = typeof value[0]
+        let object = {}
+
+        if (value.length === 0) {
+          object = {
+            key: label,
+            value: '-'
+          }
+          return this.items.push(object)
+        }
+
+        // 如果是空数组，那么循环体将不会执行
         for (const [index, item] of value.entries()) {
-          let object = {}
           if (tp === 'object') {
             const firstValue = value[0]
             if (firstValue.hasOwnProperty('name')) {
```

Comment:
It seems you've started writing code but left some parts unfinished. Please provide your questions or specifications so I can assist you better.

For instance, you could ask me to review specific sections like `Array.isArray`, etc.; optimize suggestions about certain operations; verify if the code is adhering to best practices regarding security vulnerabilities and efficiency concerns such as loops vs arrays; discuss why it's a good idea versus bad one; or validate whether there's an issue with typescript validation rules that might prevent compilation from happening correctly.